src/library/Popover/Popover.js

```diff
@@ -265,40 +265,71 @@ export default class Popover extends Component<Props, State> {
     const contentId = this.getContentId();
     const { children, disabled } = this.props;
 
+    /* console.log('Popover.getTriggerProps props', props); */
+
     const child = children ? Children.only(children) : undefined;
     const childDisabled = child && child.props.disabled !== undefined;
 
-    return {
-      ...props,
+    /* console.log(
+     *   'Popover.getTriggerProps child.props.onBlur',
+     *   child && child.props.onBlur
+     * );
+     * console.log(
+     *   'Popover.getTriggerProps child.props.onClick',
+     *   child && child.props.onClick
+     * );
+     */
+    const triggerProps = {
```

Comment:
Yep, this looks good.  I just tightened up the syntax a bit and pushed to the main branch.  https://github.com/mineral-ui/mineral-ui/blob/9b37cf6fea29d631f0917ac815f28ebf1ecdf33e/src/library/Popover/Popover.js#L263